README.md

```diff
@@ -7,11 +7,16 @@
 To start the standalone instance of Grafeas with PostgreSQL, follow these steps:
 
 1. Install [Docker Compose](https://docs.docker.com/compose/install/).
-2. Build the images and run the containers:
```

Comment:
Should this be 2?

Reply:
markdown renders this correctly, just makes it easier if these are ever re-ordered or a new step is added to not have to re-number the whole list